bash_completions.go

```diff
@@ -437,7 +444,7 @@ func writePostscript(buf io.StringWriter, name string) {
 	WriteStringAndCheck(buf, fmt.Sprintf(`if [[ $(type -t compopt) = "builtin" ]]; then
     complete -o default -F __start_%s %s
 else
-    complete -o default -o nospace -F __start_%s %s
+    complete -F __start_%s %s
```

Comment:
`compopts` should have set (-o nospace, +o default, -nosort) for us at

Reply:
This entry is for bash 3. The `nospace` is annoying but I believe it's for the `--flag=` completions. Is rather not change this old behavior.

Any reason you are changing it?

Reply:
I was of the opinion that if the `-o nospace` is required then https://github.com/spf13/cobra/blob/87cd9437e12b95810770f02ec1e55bcd774c2256/bash_completions.go#L125-L130 would have set it for any completion downstream

Reply:
I am happy reverting this change if we want to keep the legacy bash completions as is.

Reply:
So the problem with bash 3 is that `compopt` does not exist.  So we can't control those options.  That is why `nospace` was on by default.

So yeah, please revert 😄

Reply:
Reverted! Thank you :-)